jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -162,6 +162,37 @@ function with_termination_log() {
   return "$rc"
 }
 
+function with_retries() {
+  local -r \
+    retry_interval=$1 \
+    timeout_seconds=$2 \
+    cmd=("${@:3}") \
+  #---
+
+  local rc
+
+  # Use Bash built-in variable SECONDS
+  start=$SECONDS
```

Comment:
```suggestion
  start=$EPOCHSECONDS  # bash built-in
```

Reply:
For this use-case both SECONDS and EPOCHSECONDS are equally good, aren't they?

Reply:
I think so, but maybe let's take the suggestion as the second parameter is already EPOCHSECONDS